### ISSUE TITLE
account_invoice_import: add "default" import configuration

### ISSUE THREAD

**Issue by TeoGoddet** [OCA/edi#565]
We would like to be able to autocreate the partner and have a generic import configuration (independent of the partner)

This is not suitable for all invoices (old style pdf, ...) but would be very nice for some types such as https://github.com/OCA/l10n-switzerland/pull/649

**Comment by bosd** [comment]
I like the idea.
I'm using the invoice2data library a lot. Previously it was not possible to import invoices lines. With my latest pr I try to solve that.
Because it was not implemented everything had the "single line no product config".

Are all those options still needed? I mean, if there is no line data passed. Then there is nothing to import.
Maybe the generic "multiline auto product" fits all use cases?

1 When there is no line data, (there is no lines to import so continue without)
2 When there is line data it could be mapped.
3 In case there is line data, but it could not be mapped to a product. Continue the import and don't map it.

That would involve a change. As in current workflow the whole import is stopped when one product could not be mapped.
Actually in a real world scenario that is quite common. So I'll propose to change that. If a product could not be mapped, generate an warning message in the chatter of the invoice. But continue to import all the lines.

tl;dr: With some changes in code. "Multiline Auto product"  could be the onse size fits all.
